tartufo/util.py

```diff
@@ -44,29 +45,49 @@ def convert_regexes_to_rules(regexes: Dict[str, Pattern]) -> Dict[str, Rule]:
     }
 
 
-def echo_issues(
+def echo_result(
     issues: "List[Issue]",
-    as_json: bool,
+    options: "types.GlobalOptions",
+    scanner: "ScannerBase",
+    now: str,
     repo_path: str,
     output_dir: Optional[pathlib.Path],
 ) -> None:
     """Print all found issues out to the console, optionally as JSON.
 
     :param issues: The list of issues to be printed out
-    :param as_json: Whether the output should be formatted as JSON
+    :param options: Global options object
+    :param scanner: ScannerBase contaning excluded paths from config tree
+    "param now: Timestamp for the scan in isoformat (str)
     :param repo_path: The path to the repository the issues were found in
     :param output_dir: The directory that issue details were written out to
     """
-    if as_json:
+    if options.json:
         output = {
             "project_path": repo_path,
             "output_dir": str(output_dir) if output_dir else None,
+            "excluded_paths": [str(path.pattern) for path in scanner.excluded_paths],
+            "excluded_signatures": [
+                str(signature) for signature in options.exclude_signatures
+            ],
             "found_issues": [issue.as_dict() for issue in issues],
         }
         click.echo(json.dumps(output))
     else:
-        for issue in issues:
-            click.echo(issue)
+        if not issues:
+            if not options.quiet:
+                click.echo(f"Time: {now}")
+                click.echo("All clear. No secrets detected.")
```

Comment:
```suggestion
                click.echo(f"Time: {now}\nAll clear. No secrets detected.")
```
I see no reason not to keep these messages on a single line. 😄
(This is actually a micro-optimization; @jgowdy has found in some experimentation that the `click.echo` function can actually be a bit slow at times. So consolidating calls to it just seems like a good general best practice.)

Reply:
Thanks, updated this